packages/requisitions/src/ResponseRequisition/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useMemo, useState, useEffect } from 'react';
+import {
+  ArrowRightIcon,
+  useTranslation,
+  useNotification,
+  SplitButton,
+  SplitButtonOption,
+  useConfirmationModal,
+  RequisitionNodeStatus,
+} from '@openmsupply-client/common';
+import {
+  getNextResponseRequisitionStatus,
+  getStatusTranslation,
+} from '../../../utils';
+import {
+  useIsResponseRequisitionDisabled,
+  useResponseRequisitionFields,
+} from '../../api';
+
+const getStatusOptions = (
+  currentStatus: RequisitionNodeStatus,
+  getButtonLabel: (status: RequisitionNodeStatus) => string
+): SplitButtonOption<RequisitionNodeStatus>[] => {
+  const options: [
+    SplitButtonOption<RequisitionNodeStatus>,
+    SplitButtonOption<RequisitionNodeStatus>
+  ] = [
+    {
+      value: RequisitionNodeStatus.New,
+      label: getButtonLabel(RequisitionNodeStatus.Draft),
```

Comment:
there's a disconnect between the crumbs and the split button - one has `Draft > Finalised` and the other `New > Finalised` 

I'd run with Draft for both 🤷 


<img width="215" alt="Screen Shot 2022-02-22 at 9 40 04 AM" src="https://user-images.githubusercontent.com/9192912/155024198-6bf0376d-c2f1-4cf3-8588-ece22d14fe0e.png">
<img width="273" alt="Screen Shot 2022-02-22 at 9 40 10 AM" src="https://user-images.githubusercontent.com/9192912/155024211-0745b437-990f-4461-a6d3-a64ce5477978.png">